### ISSUE TITLE
Document the required statements

### ISSUE THREAD

**Issue by sabas** [yarl/gravemap#1]
In the About page there should be a section documenting the statements a person should add to make a grave appear onto the map.
(P119 +  P625; P1442)

I added some in Staglieno cemetery http://graves.wiki/#/map/@44.430447,8.950124,16z

**Comment by yarl** [comment]
Yes, it will be done. For now there is [About page](http://graves.wiki/#/about), but it says something only to people that knows SPARQL. It's also incomplete.
